system-int-test/test/cmr/system_int_test/bootstrap/fingerprint_test.clj

```diff
@@ -82,7 +83,7 @@
                     (get-in latest-revision-concept [:extra-fields :fingerprint]))))))
 
        (testing "update fingerprint of a deleted variable"
-         (let [{:keys [concept-id revision-id]} (ingest/delete-concept var2-concept)
+         (let [{:keys [_ _]} (ingest/delete-concept var2-concept)
```

Comment:
Put (ingest/delete-concept var2-concept) before the let statement 
Remove the first let statement.
The code should look like 
`
(testing "update fingerprint of a deleted variable"
  (ingest/delete-concept var2-concept)
  (let [latest-revision-concept (mdb/get-concept var2-concept-id)]
`
if you want to keep the first line in the let statement then just remove {:keys [_ _]} and replace with _